public/app/features/apiserver/server.ts

```diff
@@ -1,38 +1,81 @@
 import { config, getBackendSrv } from '@grafana/runtime';
 
-import { ListOptions, MetaStatus, Resource, ResourceForCreate, ResourceList, ResourceServer } from './types';
+import {
+  ListOptions,
+  ListOptionsLabelSelector,
+  MetaStatus,
+  Resource,
+  ResourceForCreate,
+  ResourceList,
+  ResourceServer,
+} from './types';
 
 export interface GroupVersionResource {
-  group: string;
-  version: string;
+  group?: string;
+  version?: string;
```

Comment:
when would these need to be optional?

Reply:
This was an experiment and forgot about it. I'll revert it directly on your branch.